src/main/java/api/kirana/register/transaction/models/Transaction.java

```diff
@@ -1,22 +1 @@
-package api.kirana.register.transaction.models;
 
-import lombok.Data;
-import org.springframework.data.annotation.Id;
-import org.springframework.data.mongodb.core.mapping.Document;
-
-import java.io.Serializable;
-
-
-@Data
-@Document(collection="Transaction")
-public class Transaction implements Serializable {
-    @Id
-    private String transactionId;
-    private String transactionType;
-    private String transactionStatus;
-    private double transactionAmount;
-    private String paymentMethod;
-    private String transactionTime;
```

Comment:
why do u need Time and Date in differnet fields, just save a Date it contains timestamp